account-kit/react/src/components/auth/sections/EmailAuth.tsx

```diff
@@ -16,116 +17,118 @@ type EmailAuthProps = Extract<AuthType, { type: "email" }>;
 
 // this is not used externally
 // eslint-disable-next-line jsdoc/require-jsdoc
-export const EmailAuth = ({
-  hideButton = false,
-  buttonLabel = ls.login.email.button,
-  placeholder = ls.login.email.placeholder,
-}: EmailAuthProps) => {
-  const { setAuthStep } = useAuthContext();
-  const signer = useSigner();
-  const { authenticateAsync, isPending } = useAuthenticate({
-    onMutate: async (params) => {
-      if ("email" in params) {
-        setAuthStep({ type: "email_verify", email: params.email });
-      }
-    },
-    onSuccess: () => {
-      setAuthStep({ type: "complete" });
-    },
-    onError: (error) => {
-      console.error(error);
-      setAuthStep({ type: "initial", error });
-    },
-  });
+export const EmailAuth = memo(
```

Comment:
this component is pretty expensive to render, so I'm just memoizing it because in demo app it is always present. This saves on a bunch of renders